playbook/app/pb_kits/playbook/pb_legend/_legend.jsx

```diff
@@ -18,7 +18,13 @@ type LegendProps = {
     | "data_4"
     | "data_5"
     | "data_6"
-    | "data_7",
+    | "data_7"
+    | "success"
+    | "warning"
+    | "error"
+    | "info"
+    | "neutral"
+    | "primary",
```

Comment:
can we alphabetize this list (here and in the ruby file) so that it's more readable?

Reply:
Sure we can. As far as color goes I see data_8 is missing. Do we want to add product and category colors as well? @that-jill @kre8sions .

Reply:
@JelenaD1 yes